Directory.Packages.props

```diff
@@ -11,7 +11,7 @@
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageVersion Include="Pluralize.NET" Version="1.0.2" />
     <PackageVersion Include="PowerShellStandard.Library" Version="3.0.0-preview-02" />
```

Comment:
Side note, are we stuck on this version because of PowerShell 3 support? If not let's update it to 5.1.1.

Reply:
Yes. Although it was an improvement to have as not all projects had to reference the psv3/4/5 reference assemblies. since it's just for referencing and not shipped, I don't mind TBH. Regarding dropping support, how about we start soft and just bump the minimum version in manifest and see what people's feedback is?